apps/src/weblab/WebLab.js

```diff
@@ -217,6 +244,27 @@ WebLab.prototype.onIsRunningChange = function () {
 };
 
 /**
+ * Load the asset list and store it as this.currentAssets
+ */
+WebLab.prototype.loadCurrentAssets = function () {
+  assetsApi.ajax('GET', '', (xhr) => {
+    assetListStore.reset(JSON.parse(xhr.responseText));
```

Comment:
bad things, I'm sure.
This is a copy/paste for how assetsApi is being used elsewhere. We should probably clean this up across the board. (see `AssetManager.jsx` and `Attachments.jsx`)